### PR TITLE
Add GPG to Roundcube

### PR DESCRIPTION
**Pull request by PhilRW**
The web UI was complaining that it couldn't find the GPG binary. Turns out it wasn't installed.

## What type of PR?

bug fix

## What does this PR do?

Adds GPG binary to Roundcube

### CODE REVIEW

File: webmails/roundcube/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ ENV ROUNDCUBE_URL https://github.com/roundcube/roundcubemail/releases/download/1
 RUN apt-get update && apt-get install -y \
       zlib1g-dev libzip4 libzip-dev \
       python3-jinja2 \
+      gpg
```
Comment (kaiyou) [inline webmails/roundcube/Dockerfile]: I would say this will break the build because of a missing backslash.

**Comment by kaiyou** [comment]
Otherwise sounds okay, thank you! :)

File: webmails/roundcube/Dockerfile (hunk 1 above)
Comment (PhilRW) [reply webmails/roundcube/Dockerfile]: Whoopsies. Fixed.

**Comment by Nebukadneza** [comment]
bors try

**Comment by Nebukadneza** [comment]
Darn, there’s a conflict now. Would be please be so kind? (also, sorry for our buildsystem acting up at the moment …)

**Comment by Nebukadneza** [comment]
… please exuse me for being impatient and bold, but i always wanted to try that big glaring `Resolve conflicts` button on github … wish me luck ^_^

**Comment by Nebukadneza** [comment]
bors try

**Comment by JulesTriomphe** [comment]
Any news on when this PR might be merged into master ? I would love to use the feature

**Pull request merged**